ProcessKiller/Properties/Resources.resx

```diff
@@ -136,6 +136,18 @@
     <value>Minimun matches to show "Kill all instances"</value>
   </data>
   <data name="plugin_setting_show_command_line" xml:space="preserve">
-    <value>Show command line of the process, this makes the plugin slower.</value>
+    <value>Show command line of the process in the tool tip</value>
+  </data>
+  <data name="plugin_setting_show_command_line_description" xml:space="preserve">
+    <value>This may slow down the plugin.</value>
+  </data>
+  <data name="plugin_tool_tip_path" xml:space="preserve">
+    <value>Path</value>
+  </data>
+  <data name="plugin_tool_tip_main_window" xml:space="preserve">
+    <value>Main window</value>
```

Comment:
Change to upper case
```suggestion
    <value>Main Window</value>
```

Reply:
Thought the right gramar only starts the first word with upper case. Or do you like to use TitleCase format?

Reply:
Current layout feels a bit messy

![2024-07-16 161941](https://github.com/user-attachments/assets/19ed6b4d-b701-49fc-af37-b00817261ed3)

I want to change the layout to something like Process Explorer did

![2024-07-16 162343](https://github.com/user-attachments/assets/c8f1a8cc-a3a6-4fb8-890e-b3e7959312e7)

But that can be another PR later.

Reply:
In general this is possible. But the word wrapping will be a problem.

Reply:
I just thinking of layout like this:
![image](https://github.com/user-attachments/assets/9c4c81ca-462f-4534-b7f7-bead48c476fe)
And maybe remove empty one.